connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerController.kt

```diff
@@ -110,8 +165,13 @@ internal class StripeConnectWebViewContainerController<Listener : StripeEmbedded
     fun shouldOverrideUrlLoading(context: Context, request: WebResourceRequest): Boolean {
         val url = request.url
         return if (url.host?.lowercase() in ALLOWLISTED_HOSTS) {
-            // TODO - add an analytic event here to track this unexpected behavior
             logger.warning("($loggerTag) Received pop-up for allow-listed host: $url")
+            analyticsService.track(
+                ConnectAnalyticsEvent.ClientError(
+                    error = "Unexpected pop-up",
+                    errorMessage = "Received pop-up for allow-listed host: $url"
```

Comment:
I think we should only log the URL's host in the message, rather than the entire URL. It's possible the URL could contain sensitive data (e.g. if it's a bank auth redirect, it could contain an oauth token).